MIP/mip-n/README.md

```diff
@@ -0,0 +1,53 @@
+# MIP-42: AB-FFS Governed Gas Pool
+- **Description**: Introduces a governed gas pool for AB-FFS which allows for initial period wherein used gas is not recirculated.
+- **Authors**: [Liam Monninger](mailto:liam@movementlabs.xyz)
+- **Reviewer**: Andreas Penzkofer
+- **Desiderata**: [MD-38](https://github.com/movementlabsxyz/MIP/pull/38)
+
+## Abstract
+We propose a governed gas pool both as provisional and potentially permanent solution to the supply problem described in [MD-38](https://github.com/movementlabsxyz/MIP/pull/38). This solution consists of depositing funds into a pool on the L2, and holding votes against actions for the recirculation of said funds.
+
+The concept of a governed gas pool can be applied generally to manage migrations between systems. For example, the governed gas pool could be used to transition from the "Trickle-back" mechanism referenced in [MD-38](https://github.com/movementlabsxyz/MIP/pull/38) to [MIP-42](https://github.com/movementlabsxyz/MIP/pull/42), as is used as an example amongst others herein.
+
+## Motivation
+The AB-FFS under fixed supply presents several challenges presented in [MD-38](https://github.com/movementlabsxyz/MIP/pull/38). This MIP seeks to address the question of where used gas funds should be deposited with a simple and flexible solution. By simply depositing funds into a pool and holding votes against actions for the recirculation of said funds, we can create a system that can provide a short-term solution to the supply problem and generalize to other systems. 
+
+## Specification
+![Governed Gas Pool](governed-gas-pool.png)
+
+1. Gas used on the L2 MUST be deposited into the Governed Gas Pool.
+2. The Governed Gas Pool MUST disperse deposited funds according to a proposal made by Proposers which is enacted by a vote of Executors.
+3. The Governed Gas Pool SHOULD define a maximum emissions rate for the pool s.t. the governing body cannot engage in Byzantine or other malicious behavior.
```

Comment:
```suggestion
3. The Governed Gas Pool SHOULD define a maximum emissions rate for the pool s.t. the governing body cannot engage in Byzantine or other malicious behavior.
4. Governance MUST be implemented using the [aptos_governance](https://github.com/movementlabsxyz/aptos-core/blob/andyjsbell/atomic-bridge/aptos-move/framework/aptos-framework/sources/aptos_governance.move) module provided by aptos core. 
5. Any necessary upgrades or migrations required to `aptos_framework` MUST be included within a proposal and voted upon by the governing body. 
```

Reply:
Governed gas pool is not specific to Aptos that would be a separate MIP.

Reply:
Ok, in that case I'll put the reference implementation there also, as the ref implementation I wrote assumes usage of `aptos_governance.move`.